### PR TITLE
[#169626834] Added Markdown list in SPID/CIE Information screen

### PR DESCRIPTION
**Pull request by alex193a**
Added the Markdown list in the SPID/CIE Information screen

### CODE REVIEW

**Comment by digitalcitizenship** [comment]
## Affected stories

  * 🌟 [#169626834](https://www.pivotaltracker.com/story/show/169626834): Utilizzare lista Markdown nella schermata di informazioni SPID/Cie

<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: locales/it/spid.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+**SPID**, il Sistema Pubblico di Identità Digitale, ti permette di accedere a tutti i servizi della Pubblica Amministrazione con un'unica Identità Digitale (username e password). [Per saperne di più](https://www.cartaidentita.interno.gov.it)
```
Comment (Undermaken) [inline locales/it/spid.md]:
```suggestion
**SPID**, il Sistema Pubblico di Identità Digitale, ti permette di accedere a tutti i servizi della Pubblica Amministrazione con un'unica Identità Digitale (username e password). [Per saperne di più](https://www.spid.gov.it/)
```

File: locales/en/index.yml [hunk 2]
```diff
@@ -233,6 +233,11 @@ authentication:
     unknownCardTitle: Unknown card
     unknownCardContent: The card used does not seem to be an Electronic Identity Card (CIE).
     unknownCardHelpLink: What is the Electronic Identity Card?
+  cie_information_request: !include cie.md
+  spid_or_cie: To use IO you must have one of these digital tools, which certify your identity.
+  request_cie: Request EIC
```
[the remark this answers is not in the document]
Comment (Alex193a) [reply locales/en/index.yml]: I've checked it and it seems to be correctly aligned with the bottom of the number.
Schermata 2019-11-19 alle 14 45 47

**Comment by Undermaken** [comment]
Since we merged this [PR](https://github.com/teamdigitale/io-app/pull/1383) there are some conflicts to solve

**Pull request closed without merging**